javascript/elements/updates_for_element.js

```diff
@@ -99,6 +97,43 @@ export default class UpdatesForElement extends SubscribingElement {
     }
   }
 
+  async resolveTurboFrames (documentFragment) {
+    const reloadingTurboFrames = [
+      ...documentFragment.querySelectorAll(
+        'turbo-frame[src]:not([loading="lazy"])'
```

Comment:
I'm still puzzled that Turbo Frames puts `loading="lazy"` only on non-lazy instances. What am I not understanding? I trust that it works, it's just so bizarre.

Reply:
I know it's blasphemous to ask this from _you_, but refer to the docs: 

https://turbo.hotwired.dev/reference/frames#eager-loaded-frame
https://turbo.hotwired.dev/reference/frames#lazy-loaded-frame

the distinction between them is that the latter uses an IntersectionObserver, whereas the former doesn't.

So my reasoning was, when something shouldn't be loaded until it's visible, it's probably best to follow along in this behavior.

Reply:
That's a great answer! Thank you.

I admit that my eyes glazed over the distinction.